packages/js/number/src/index.ts

```diff
@@ -1,13 +1,21 @@
-const numberFormatter = require( 'locutus/php/strings/number_format' );
+/**
+ * External dependencies
+ */
+import numberFormatter from 'locutus/php/strings/number_format';
 
 /**
  * Number formatting configuration object
  *
  * @typedef {Object} NumberConfig
- * @property {number} [precision]         Decimal precision.
- * @property {string} [decimalSeparator]  Decimal separator.
- * @property {string} [thousandSeparator] Character used to separate thousands groups.
+ * @property {number|string|null} [precision]         Decimal precision.
```

Comment:
Test cases show `precision` value can be a number, string or null.

https://github.com/woocommerce/woocommerce/blob/323a78a4d785246ded1085a8b0e62f2cf126ce17/packages/js/number/src/test/index.js#L13-L50